tests/test_end_2_end.py

```diff
@@ -1,5 +1,9 @@
 import pytest
+from testix import *
 import subprocess
+import crawler.crawl as crawl
+import crawler.links as links
```

Comment:
import modules, not names (see [here](https://github.com/PracticeFoxyCode/practice#namespaces-and-import-statements))